v4/src/IokText.jsx

```diff
@@ -92,7 +93,7 @@ class IokText extends Component {
 
   downloadGraph() {
     const { cy } = this.state;
-    const exportObj = getNodesEdgesJson(cy);
```

Comment:
Re your comment on graph formats offline @rianadon. This change updates v4 webapp to import and export "cytoscape format" graphs of the form `elements: {nodes, edges}`